packages/expo-updates/src/UpdatesProvider.types.ts

```diff
@@ -0,0 +1,123 @@
+import type { UpdatesProviderEventType } from './UpdatesProvider.constants';
+
+/**
+ * An event passed into the optional `providerEventHandler` parameter of [`useUpdates()`](#useupdatesprovidereventhandler).
+ * Updates.Provider methods [`checkForUpdate()`](#checkforupdate), [`downloadUpdate()`](#downloadupdate),
+ * or [`downloadAndRunUpdate()`](#downloadandrunupdate), all make calls to the event handler.
+ */
+export type UpdatesProviderEvent = {
+  /**
+   * One of the possible provider event types.
+   */
+  type: UpdatesProviderEventType;
+  /**
+   * For event types associated with errors, this will contain the error object that caused the event.
+   */
+  error?: Error;
+};
+
+/**
+ * Structure encapsulating information on the currently running app
+ * (either the embedded bundle or a downloaded update).
+ */
+export type CurrentlyRunningInfo = {
+  /**
+   * The UUID that uniquely identifies the currently running update if `expo-updates` is enabled. The
+   * UUID is represented in its canonical string form (`xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`) and
+   * will always use lowercase letters. In development mode, or any other environment in which
+   * `expo-updates` is disabled, this value is `null`.
+   */
+  updateId: string | null;
+  /**
+   * The channel name of the current build, if configured for use with EAS Update. Null otherwise.
+   */
+  channel: string | null;
+  /**
+   * If `expo-updates` is enabled, this is a `Date` object representing the creation time of the update
+   * that's currently running (whether it was embedded or downloaded at runtime).
+   *
+   * In development mode, or any other environment in which `expo-updates` is disabled, this value is
+   * null.
+   */
+  createdAt: Date | null;
+  /**
+   * This will be true if the currently running update is the one embedded in the build,
+   * and not one downloaded from the updates server.
+   */
+  isEmbeddedLaunch: boolean;
+  /**
+   * `expo-updates` does its very best to always launch monotonically newer versions of your app so
+   * you don't need to worry about backwards compatibility when you put out an update. In very rare
+   * cases, it's possible that `expo-updates` may need to fall back to the update that's embedded in
+   * the app binary, even after newer updates have been downloaded and run (an "emergency launch").
+   * This boolean will be `true` if the app is launching under this fallback mechanism and `false`
+   * otherwise. If you are concerned about backwards compatibility of future updates to your app, you
+   * can use this constant to provide special behavior for this rare case.
+   */
+  isEmergencyLaunch: boolean;
+  /**
+   * If `expo-updates` is enabled, this is the
+   * [manifest](/workflow/expo-go#manifest) object for the update that's currently
+   * running.
+   *
+   * In development mode, or any other environment in which `expo-updates` is disabled, this object is
+   * empty.
+   */
+  manifest: Partial<Manifest>;
```

Comment:
```suggestion
  manifest: Manifest;
```

Reply:
Done

Reply:
Ah, sorry, I meant to only add this comment to `AvailableUpdateInfo.manifest`, my mad